lib/cocoapods/installer/xcode/multi_pods_project_generator.rb

```diff
@@ -13,7 +13,9 @@ def generate!
           # Generate container Pods.xcodeproj.
           container_project = create_container_project(aggregate_targets, sandbox.project_path)
 
-          project_paths_by_pod_targets = pod_targets.group_by { |pod_target| sandbox.pod_target_project_path(pod_target.pod_name) }
+          project_paths_by_pod_targets = pod_targets.group_by do |pod_target|
+            sandbox.pod_target_project_path(pod_target.project_name)
+          end
```

Comment:
Ah we run into another fun little problem with split projects: `def pod_name_from_grouping` relies upon `pod_name` to determine if the pod is local or not. This gates the behavior of putting the subproject into the `Development Pods` or `Pods` group. My opinion is just to put it in the `Pods` group and only put it in the `Development Pods` group if all are local. What do you think?

Reply:
Answered below.